content/en/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -59,12 +59,12 @@ Kubespray provides the ability to customize many aspects of the deployment:
 * Choice deployment mode: kubeadm or non-kubeadm
 * CNI (networking) plugins
 * DNS configuration
-* Choice of control plane: native/binary or containerized with docker or rkt
+* Choice of control plane: native/binary or containerized with docker 
```

Comment:
Not sure to keep only "or containerized" or explain which CRI is available

Reply:
Agreed "or containerized" would be more flexible